### ISSUE TITLE
Unable to find icon with name ‘globe’ in icon set ‘faicon’

### ISSUE THREAD

**Issue by stardiviner** [rainstormstudio/nerd-icons.el#9]
When I execute elisp code:
```elisp
(nerd-icons-icon-for-url "http://www.cyberciti.biz/faq/bash-for-loop/")
```
I got following error:
```
Debugger entered--Lisp error: (error "Unable to find icon with name ‘globe’ in icon set ‘faicon’")
  signal(error ("Unable to find icon with name ‘globe’ in icon set ..."))
  error("Unable to find icon with name `%s' in icon set `%s..." "globe" faicon)
  nerd-icons-faicon("globe")
  apply(nerd-icons-faicon "globe")
  #f(compiled-function (url &rest arg-overrides) "Get the formatted icon for URL.\nIf an icon for URL isn't found in `nerd-icons-url-alist', a globe is used.\nARG-OVERRIDES should be a plist containining `:height',\n`:v-adjust' or `:face' properties like in the normal icon\ninserting functions." #<bytecode -0x196d01fc7f714d17>)("http://www.cyberciti.biz/faq/bash-for-loop/")
  apply(#f(compiled-function (url &rest arg-overrides) "Get the formatted icon for URL.\nIf an icon for URL isn't found in `nerd-icons-url-alist', a globe is used.\nARG-OVERRIDES should be a plist containining `:height',\n`:v-adjust' or `:face' properties like in the normal icon\ninserting functions." #<bytecode -0x196d01fc7f714d17>) "http://www.cyberciti.biz/faq/bash-for-loop/")
  nerd-icons-icon-for-url("http://www.cyberciti.biz/faq/bash-for-loop/")
  eval((nerd-icons-icon-for-url "http://www.cyberciti.biz/faq/bash-for-loop/") t)
  eval-expression((nerd-icons-icon-for-url "http://www.cyberciti.biz/faq/bash-for-loop/") nil nil 127)
  funcall-interactively(eval-expression (nerd-icons-icon-for-url "http://www.cyberciti.biz/faq/bash-for-loop/") nil nil 127)
  call-interactively(eval-expression nil nil)
  command-execute(eval-expression)
```

**Comment by rainstormstudio** [comment]
It's fixed now: 0e2fe5bafd324540cda6745f8c91e0728beff32c

**Comment by stardiviner** [comment]
@rainstormstudio Thanks for quick fix.
